istio.VERSION

```diff
@@ -4,6 +4,6 @@ export CA_HUB="docker.io/istio"
 export CA_TAG="2017-04-13-17.43.08"
 export MIXER_HUB="docker.io/istio"
 export MIXER_TAG="2017-05-09-02.32.46"
-export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/dbcc933388561cdf06cbe6d6e1076b410e4433e0/artifacts/istioctl"
+export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/1cb69d3458079c798bd87ec54f785725246007bc/artifacts/istioctl"
```

Comment:
OK. We need a new ISTIOCTL_URL with the latest fix. I just found one that worked. Do you know what the latest in master is?